fast-af/fast-checkout-woocommerce
# PINT-477 - Display plugin version

# Description

Add a footer to the Fast settings page in the WP admin that includes the Fast version number along with important links, including the following:

- [Version #](https://www.wordpress.org/plugins/fast-checkout-for-woocommerce)
- [Seller Login](https://fast.co/business)
- [About](https://fast.co/home)
- [Help](https://help.fast.co/)
- [Terms](https://www.fast.co/terms/terms-home)
- [Privacy](https://www.fast.co/privacy)

# Testing instructions

- Login to the [staging admin](https://fasttestdev.wpcomstaging.com/wp-admin/).
- Verify that the updated footer is displaying properly.
- Verify that each item in the footer links to where it should.

![fast-settings-footer-no-bg-black-text](https://user-images.githubusercontent.com/636415/124988808-16168e00-e004-11eb-8a3a-27a393908e6a.png)

# Checks

@brikr @ilkerulutas this is ready for review
